de.flinkebits.AvaEmailArchivar.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>de.flinkebits.AvaEmailArchivar</id>
+    <launchable type="desktop-id">de.flinkebits.AvaEmailArchivar.desktop</launchable>
+    <metadata_license>LicenseRef-proprietary</metadata_license>
```

Comment:
this will not pass validation. If you wrote that file, please license it CC0-1.0. Otherwise if that come form upstream that way we can't even use it 

(and then if it comes from upstream it shouldn't be here anyway)